joern-cli/frontends/c2cpg/src/main/scala/io/joern/c2cpg/parser/CdtParser.scala

```diff
@@ -102,6 +117,14 @@ class CdtParser(config: Config) extends ParseProblemsLogger with PreprocessorSta
     parse(file).map(t => preprocessorStatements(t)).getOrElse(Iterable.empty)
   }
 
+  def parse(code: String, inFile: Path): Option[IASTTranslationUnit] = Try(parseInternal(code, inFile)) match {
```

Comment:
scalafmt should really handle this automatically, but for readability and consistency can you please add a newline after the method declaration, and since it's rather long also some curly braces?